app/views/idv/capture_doc/front_image.html.erb

```diff
@@ -0,0 +1,34 @@
+<% title t('doc_auth.titles.doc_auth') %>
+
+<%= render 'idv/doc_auth/error_messages', flow_session: flow_session %>
+
+<%= render 'idv/doc_auth/front_of_state_id_image' %>
+
+<h1 class="h3 my0">
+  <%= t('doc_auth.headings.upload_front') %>
+</h1>
+
+<%= t('doc_auth.info.upload_image') %>
+
+<%= validated_form_for(:doc_auth, url: idv_capture_doc_step_path(step: :front_image), method: 'PUT',
+                       html: { autocomplete: 'off', role: 'form', class: 'mt2' }) do |f| %>
+  <%= f.input :image_data_url, as: :hidden %>
+  <div class="clearfix mxn1">
+    <div class="sm-col sm-col-8 px1 mt2">
+      <%= f.input :image, label: false, as: 'file', required: true,
+                  input_html: { aria: { invalid: false } } %>
+    </div>
+  </div>
+  <%= render 'idv/doc_auth/notices', flow_session: flow_session %>
+
+  <div class="mb4" id= "target">
+  </div>
```

Comment:
1. can we remove the space after `id=`
2. let's just put the closing tag on the same line?

```suggestion
  <div class="mb4" id="target"></div>
```